workitem/workitem_repository.go

```diff
@@ -600,7 +600,28 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, spaceID uuid.UUID,
 	if !wiType.CanConstruct {
 		return nil, errors.NewForbiddenError(fmt.Sprintf("cannot construct work items from \"%s\" (%s)", wiType.Name, wiType.ID))
 	}
-
+	var exists bool
+	// Prohibit creation of work items from a type that doesn't belong to current space template
+	query := fmt.Sprintf(`
+		SELECT EXISTS (
+			SELECT 1 from %s WHERE id=$1 AND space_template_id = (
+				SELECT space_template_id FROM spaces WHERE id=$2
```

Comment:
Please do this:

```go
	query := fmt.Sprintf(`
		SELECT EXISTS (
			SELECT 1 from %[1]s WHERE id=$1 AND space_template_id = (
				SELECT space_template_id FROM %[2]s WHERE id=$2
			)
		)`, wiType.TableName(), space.Space{}.TableName())
```

Notice that the table name for spaces is now also substituted. Sorry if I didn't catch that before.

Reply:
Please see https://github.com/fabric8-services/fabric8-wit/pull/2115/commits/8f1fe48edd857d58db5b5fc51b067fdfa0d07057